zerver/openapi/zulip.yaml

```diff
@@ -6618,6 +6621,18 @@ paths:
             type: boolean
             default: false
           example: true
+        - name: message_ids
+          in: query
+          description: |
+            A list of message IDs to fetch. If provided, `anchor`, `num_before`,
+            and `num_after` are ignored.
```

Comment:
If `message_ids` is provided, then those other parameters should not just be ignored but be an error.

Also `include_anchor` should get the same treatment as those three — it's an error to provide it at the same time as `message_ids`.

Reply:
The code seems to do that, so this documentation is wrong.

Reply:
Oops, updated that comment to explain things in more detail.